src/microbe_stage/gui/PatchDetailsPanel.cs

```diff
@@ -154,6 +154,12 @@ public class PatchDetailsPanel : PanelContainer
 
     public bool IsPatchMoveValid { get; set; }
 
+    public bool MoveToPatchButtonVisible
+    {
+        get => moveToPatchButton.Visible;
+        set => moveToPatchButton.Visible = value;
```

Comment:
I recall there is an HSplit? It should be hidden too.

Reply:
You are correct, I will adjust it accordingly